README.md

```diff
@@ -225,6 +225,8 @@ make test-race
 
 1. Bring up KinD cluster and deploy ingress node firewall operator from the steps outlined previously.
 2. Run full E2E test
+
+Note: SCTP tests are disabled by default. Enable by setting environment variable `ENABLE_SCTP_TESTS=true`
```

Comment:
since u already editing this we should callout all args to our e2e so user of the repo can know what are the different knobs
also u need to default the same for our BM ci run here 
[openshift-ci](https://github.com/openshift/ingress-node-firewall/tree/master/openshift-ci)/run_e2e.sh ?

Reply:
also pls explain why the default for SCTP is disabled, and include lkm dependencies

Reply:
>also u need to default the same for our BM ci run here
[openshift-ci](https://github.com/openshift/ingress-node-firewall/tree/master/openshift-ci)/run_e2e.sh ?

You talking about wanting to enable SCTP tests on BM?

Reply:
Yes

Reply:
Unable to do it easily - SCTP module unloaded on CoreOS by default. Its loaded by default in the KinD image however and its tested there and works ok. Is this sufficient?